style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
+    width: 100%;
+    height: 448px;
+    transition: all 0.8s ease-in-out;
+  }
+
+  .work-card-4 .desktop-img-4:hover {
+    transform: scale(1.05);
+  }
+
+  .work-card-4 .work-detials {
+    margin-bottom: 10rem;
+    width: 50%;
+  }
+
+  .work-card-4 .work-detials .h2-desktop {
+    display: flex;
+    width: 496px;
+    height: 46px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-left: 12px;
+  }
+
+  .work-card-4 .work-detials .align-desktop {
+    display: block;
+  }
+
+  .work-card-4 .work-detials .align-desktop ul {
+    display: flex;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    margin-left: 10px;
+  }
+
+  .work-card-4 .work-detials .align-desktop ul span {
+    width: 26px;
+    height: 24px;
+    margin-right: 1.5rem;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+    padding-left: 5px;
+  }
+
+  .work-card-4 .work-detials .align-desktop ul li {
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #6b778c;
+    margin-left: 1.5rem;
+  }
+
+  .work-card-4 .work-detials .p-desktop {
+    width: 90%;
+    height: 50px;
+    display: flex;
+    flex-wrap: wrap;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    padding-left: 5px;
+    margin-left: 10px;
+  }
+
+  .work-card-4 .work-detials .tags-desktop {
+    display: block;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    margin-left: 10px;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    gap: 1rem;
+    margin-top: 2rem;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .HTML {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .Ruby-on-rails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 110px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .CSS {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```

Comment:
- [Optional] Consider using DRY principle.
```css
.flex-row-center {
    display: flex;
    flex-direction: row;
    align-items: center;
}
```
Then add the `flex-row-center` class to all elements that require these properties.